Patches/Core/PawnCapacityDefs/PawnCapacities.xml

```diff
@@ -2,17 +2,9 @@
 <Patch>
 
   <Operation Class="PatchOperationAdd">
-    <xpath>*/PawnCapacityDef[defName="BloodFiltration"]</xpath>
-    <value>
-      <labelMechanoids>capacitance</labelMechanoids>
-      <lethalMechanoids>true</lethalMechanoids>
-    </value>
-  </Operation>
-
-  <Operation Class="PatchOperationAdd">
-    <xpath>*/PawnCapacityDef[defName="BloodPumping"]</xpath>
+    <xpath>*/PawnCapacityDef[defName="Metabolism"]</xpath>
     <value>
-      <labelMechanoids>coolant pumping</labelMechanoids>
+      <labelMechanoids>coolant reservoir</labelMechanoids>
```

Comment:
Why?

Reply:
Vanilla added a number of mechanoid that conflict with the existing CE ones. I believe they made the heart coolant pumping instead of energy generation. The reservoir was the closest I could think of to have all systems represented in a way that made sense with the new vanilla changes.

Reply:
Fair enough, these are mostly just fluff anyways